src/components/editor/Editor.tsx

```diff
@@ -49,7 +52,8 @@ const Editor: React.FC<{}> = () => {
   if (structureDefinition) {
     return (
       <div>
-        <h1>Profile Editor for {structureDefinition.type}</h1>
+        <Navbar />
+        {/* <h1>Profile Editor for {structureDefinition.type}</h1>
```

Comment:
do we want to keep the commented code ?

Reply:
This will be resolved when adding ui later

Reply:
I added UI so it's not commented anymore